portfolio/src/main/webapp/lib/typed.js

```diff
@@ -0,0 +1,11 @@
+/*!
+ * 
+ *   typed.js - A JavaScript Typing Animation Library
```

Comment:
Prefer to use CDN's, reduces the code duplication and maintenance effort

Reply:
<script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>

Reply:
Removed the script and added link to the CDN.